tools/local-sensor/main.go

```diff
@@ -243,9 +250,24 @@ type sensorMessagesJSONOutput struct {
 	MessagesFromSensor []*central.MsgFromSensor `json:"messages_from_sensor"`
 }
 
-func dumpMessages(messages []*central.MsgFromSensor, start, end time.Time, outfile string) {
+func dumpMessages(messages []*central.MsgFromSensor, start, end time.Time, outfile string, binaryOutput bool) {
 	dateFormat := "02.01.15 11:06:39"
 	log.Printf("Dumping all sensor messages to file: %s\n", outfile)
+	if binaryOutput {
```

Comment:
I think it's better to do only one at a time with the `-format` param as I mentioned before. Unless there is a need to output both files for the same scenario.

Reply:
The only reason for having both at the same time (that I can think of) is in case you want to use that file in a tests but you also want to check that the file contains something. In this scenario it would be nice to have both. What do you think? Should we maybe default in both and if the user specifies `-format` then just write the specified format?

Reply:
> Should we maybe default in both and if the user specifies -format then just write the specified format?

I don't get this suggestion.

Reply:
Well, since you're writing the assertions for this test, I trust your judgment if you think it's going to be useful to have a sibling JSON file with human-readeble content. I don't have a strong opinion tbh.

Reply:
I think you're right. I've changed the code to just write the file in the specified format. Let me know what you think.